ServiceBusQueuePeekLockAbandon/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+    "title": "Service Bus Peek Lock Pattern - get message from a queue",
+    "description": "Get new message from a Service Bus queue and lock it for processing. The message will be deleted from the queue after the processing is completed. This pattern is useful when you want to ensure that the message is processed only once and not lost in case of a failure during processing.",
+    "prerequisites": "Before you can use this template, you need to configure a Service Bus namespace and create a queue. For more information about Service Bus see [Create a new queue on Service Bus](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal)",
+    "skus": [
+        "standard"
+    ],
+    "kinds": [
+        "stateful",
+        "stateless"
+    ],
+    "details": {
+        "By": "Microsoft",
+        "Type": "Workflow",
+        "Trigger": "Event"
+    },
+    "artifacts": [
+        {
+            "type": "workflow",
+            "file": "workflow.json"
+        }
+    ],
+    "images": {
+        "light": "snapshot_light",
+        "dark": "snapshot_dark"
+    },
+    "parameters": [
+        {
+            "name": "queueName",
```

Comment:
All parameter names and connection names are supposed to end with _#workflowname#, this is to ensure no duplicate parameters since it's shared across workflows under the same resource.

`queueName_#workflowname#`, `messageBatchSize_#workflowname#`

Reply:
@wsilveiranz can you add Category in details. "Category" : "Design Patterns". Your manifest validation will fail when you do so, Elain is aware of this and is working on the fix

Reply:
@divyaswarnkar I have fixed the validation earlier, and Category currently only checks whether it is a string or not as an optional property. The validation shouldn't fail due to Category.